core/controllers/reader.py

```diff
@@ -608,7 +608,9 @@ def post(self, exploration_id):
                     learner_progress_services.mark_topic_as_partially_learnt(
                         user_id, story.corresponding_topic_id)
             else:
-                raise Exception('Story with id %s does not exist.' % story_id)
+                logging.error('Could not find a story corresponding to id.')
```

Comment:
State which ID (in the error message). You can keep the same error message as before.

Reply:
Done.

> Thanks! Just some minor notes.
> 
> I worry a little bit about the name "mark_story_as_incomplete()" if it's not actually doing that if the story is already complete. Perhaps record_story_started() and record_topic_started() might be better.

Done.